golang/cosmos/app/app.go

```diff
@@ -790,13 +790,18 @@ func NewAgoricApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 
+	const (
+		upgradeName     = "agoric-upgrade-12"
+		upgradeNameTest = "agorictest-upgrade-12"
+	)
+	
```

Comment:
Not a golang specialist, but I think `const` are preferred declared top of a module